### PR TITLE
feat(curriculum): add gradebook app lab

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-gradebook-app/66bb6a9c2dd58b73cd759034.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+You should have a function named `getAverage`.
+
+```js
+assert.isFunction(getAverage);
+```
+
+Your `getAverage` function should return a number.
+
+```js
+assert.strictEqual(typeof getAverage([92, 88, 12, 77, 57, 100, 67, 38, 97, 89]), 'number');
```
Comment (jdwilkin4) [inline curriculum/challenges/english/25-front-end-development/lab-gradebook-app/66bb6a9c2dd58b73cd759034.md:47]: we should use isNumber

```suggestion
assert.isNumber(getAverage([92, 88, 12, 77, 57, 100, 67, 38, 97, 89]));
```

File: curriculum/challenges/english/25-front-end-development/lab-gradebook-app/66bb6a9c2dd58b73cd759034.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+You should have a function named `hasPassingGrade`.
+
+```js
+assert.isFunction(hasPassingGrade);
+```
+
+Your `hasPassingGrade` function should return a boolean value.
+
+```js
+assert.strictEqual(typeof hasPassingGrade(100), "boolean");
```
Comment (jdwilkin4) [inline curriculum/challenges/english/25-front-end-development/lab-gradebook-app/66bb6a9c2dd58b73cd759034.md:139]: we should use  isBoolean here

```suggestion
assert.isBoolean(hasPassingGrade(100));
```

**Review by jdwilkin4 — COMMENTED** [review]
Left two small comments but otherwise good 👍

**Review by jdwilkin4 — APPROVED** [review]
LGTM 🚀

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

Pretty fun 👍

**Pull request merged**